examples/pie_and_polar_charts/nested_pie.py

```diff
@@ -0,0 +1,63 @@
+"""
+Nested pie charts
+=================
+
+The following examples show how two ways to build a nested pie chart
+in Matplotlib.
+
+The most straightforward way to build a pie chart is to use the
+:func:`pie method <matplotlib.pyplot.pie>`:
+"""
+
+from matplotlib import pyplot as plt
+import numpy as np
+
+vals = np.array([[1, 2, 3, 4], [2, 3, 4, 5], [3, 4, 5, 6]])
+fig, ax = plt.subplots()
+ax.pie(vals.flatten(), radius=1.2,
+       colors=plt.rcParams["axes.prop_cycle"].by_key()["color"][:vals.shape[1]])
+ax.pie(vals.sum(axis=1), radius=1)
+ax.set(aspect="equal", title='Pie plot with `ax.pie`')
+
+plt.show()
+
+###############################################################################
+# However, you can accomplish the same output by using a bar plot on
+# axes with a polar coordinate system. This may give more flexibility on
+# the exact design of the plot.
```

Comment:
I think the two examples should show the same data (if not necessarily the same style).  For the one below, that probably means calling something like `np.histogram(data, cumulative=True) * 2 * np.pi` and fiddling with partial sums.